Gem/Code/Source/Platform/Linux/Utils_Linux.cpp

```diff
@@ -20,7 +22,43 @@ namespace AtomSampleViewer
 
         bool RunDiffTool(const AZStd::string& filePathA, const AZStd::string& filePathB)
         {
-            return false;
+            bool result = true;
+            AZStd::string executablePath = "/usr/bin/bcompare";
+
+            // Fork a process to run Beyond Compare app
+            pid_t childPid = fork();
```

Comment:
why is forking necessary to launch another app?

Reply:
That's the UNIX way. Pretty standard stuff, and it's the same code as in MacOS.